vta/tutorials/frontend/deploy_classification.py

```diff
@@ -63,6 +63,8 @@
 
 # Make sure that TVM was compiled with RPC=1
 assert tvm.runtime.enabled("rpc")
+# Make sure vta is available
+assert tvm.get_global_func("tvm.contrib.vta.init", True) is not None
```

Comment:
This function is not available in simulator and only available in rpc server to initialize the remote bitstream

Reply:
What function can I check then?

Reply:
I believe https://github.com/apache/tvm/pull/7761 already checks if vta binary is available